tests/integration/test_membership.py

```diff
@@ -126,18 +126,6 @@ def test_full_cluster_remove(cluster):
         expected_nodes -= 1
         leader.wait_for_num_nodes(expected_nodes)
 
-    # make sure other nodes are down
```

Comment:
@tezc As we do plan to restore the shutdown behavior in the future, how about keeping this code disabled and leave a comment that reminds us to reinstate it when shutdown is implemented?

Reply:
Done.